.github/workflows/ci.yml

```diff
@@ -40,4 +40,4 @@ jobs:
         # For example: adding multiple test runs with different parameters
         # Remember that you can parallelise this by using strategy.matrix
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
+          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results --n_haplotypes 12
```

Comment:
Tests should run with just the test profile. Add this parameter to the test.config

Reply:
@nf-core/core `--n_haplotypes` is a mandatory argument in this pipeline, because one can't reliably interfere the number of haplotypes from the given input FASTA. Of course I could set this to a default value, but then lots of users will miss to set this parameter in the first place.
Any comments from your side?

Reply:
you can add the value required for the tests in `conf/test.config`, and that doesn't override the pipeline defaults :)

Reply:
Yes! I think @FriederikeHanssen had some different question in mind? @mirpedrol .

Reply:
After personal conversation with @mirpedrol I am solving this.

Reply:
No exactly what I meant. This should go into the test.config, so the simple test are minimal. I think we misunderstood each other yesterday